docs/charts.md

```diff
@@ -229,6 +229,38 @@ Managing charts with `requirements.yaml` is a good way to easily keep
 charts updated, and also share requirements information throughout a
 team.
 
+#### Alias field in requirements.yaml
+
+In addition to the other fields above, each requirements entry may contain
+the optional field `alias`.
+
+Adding an alias for a dependency chart would add another copy
```

Comment:
I also thought about this but was fine with it. If you'd like to, please feel free to open an issue and tag me in it so we can have further discussion